app/templates/identity/api.hbs

```diff
@@ -0,0 +1,153 @@
+<h1>API</h1>
+<p>To render the Global Navigation and the Global Footer you need to:</p>
+<ul>
+	<li>{{#link-to 'overview.quick-start'}}Install Design System{{/link-to}}</li>
+	<li>Install localization messages</li>
```

Comment:
It's described in the "Localization" section. Unfortunately we can't add anchor links when we use `locationType: hash` and it's required for Github pages.